src/Components/ProtectedRoute.js

```diff
@@ -0,0 +1,39 @@
+// import React from 'react';
+// import { Redirect, Route } from 'react-router-dom';
+
+// const ProtectedRoute = ({ component: Component, ...rest }) => {
+//   const token = localStorage.getItem('token');
+//   const isAuthenticated = !!token;
```

Comment:
This Part is Commented...You can remove this commented part if not required in future

Reply:
ok

Reply:
I have resolved.

Reply:
Changes Accepted